File: Android/AbuMonitor/mobile/src/main/java/com/example/comunicationwearmobile/Interface/RouteMaker.java
package com.example.comunicationwearmobile.Interface;

import com.example.comunicationwearmobile.models.maps.JsonRoutePojo;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Url;

public interface RouteMaker {
    @GET
    Call<JsonRoutePojo> getAllWayPoints(@Url String url);
}

File: Android/AbuMonitor/mobile/src/main/java/com/example/comunicationwearmobile/Interface/RouteMaker.kt
package com.example.comunicationwearmobile.Interface

import com.example.comunicationwearmobile.models.maps.RouteApiOSRMJson
import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Url

interface RouteMaker {
    @GET
    fun getAllWayPoints(@Url url: String?): Call<RouteApiOSRMJson?>?
}
